### PR TITLE
Daily prod issue fix: invalid HTML URLs

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

It was found during daily upstream productization that some HTML links are no longer pointing to valid resources. This is being fixed.

#### Rationale:

- keeping the content helpful means that links work

#### Review Hints:

see commits and try to access old and new URLs

### CODE REVIEW

**Review by jan-cerny — CHANGES_REQUESTED** [review]
@vojtapolasek I confirm all 3 new links work fine. But, I have found that are multiple other occurrences of `https://www.iso.org/standard/54534.html` in our project. I think that they need to be changed as well.

**Comment by vojtapolasek** [comment]
@jan-cerny  you are right, I overlooked these. Now they should be fixed.

**Review by jan-cerny — APPROVED** [review]

**Comment by ggbecker** [comment]
@ComplianceAsCode/ubuntu-maintainers @ComplianceAsCode/suse-maintainers @ComplianceAsCode/oracle-maintainers 

I am overriding the need of your approvals since this is not really affecting products other than having wrong URLs in the project.

**Pull request merged**
